src/components/SlateEditor/plugins/unsupported/unsupportedPlugin.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2025-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { createPlugin, isElementOfType, PARAGRAPH_ELEMENT_TYPE, SPAN_ELEMENT_TYPE } from "@ndla/editor";
+import { UNSUPPORTED_ELEMENT_TYPE, UNSUPPORTED_PLUGIN } from "./types";
+import { Editor, Element, Node, Transforms } from "slate";
+import { blockContentToHTML } from "../../../../util/articleContentConverter";
+
+export const unsupportedPlugin = createPlugin({
+  name: UNSUPPORTED_PLUGIN,
+  type: UNSUPPORTED_ELEMENT_TYPE,
+  normalize: (editor, node, path, logger) => {
+    if (!Element.isElement(node) || node.type === UNSUPPORTED_ELEMENT_TYPE) return false;
+    if (editor.supportsElement(node)) return false;
+    logger.log(`Encountered an element that no plugin supports, converting to unsupported element, ${node.type}`);
```

Comment:
Betyr dette at serializer må ligge til slutt i lista? Eller er det pluginen som må ligge til slutt i pluginliste. I tilfelle det siste er den ikkje det i commentToolbarUtils.

Reply:
Nei, såvidt jeg vet kan denne ligge hvor som helst. `editor.supportsElement` har ingenting å gjøre med plassering i listen.

Reply:
Tenkte meir på om serialiseringa må ligge sist for å være sikker på at ingen andre plugins har tatt tak i elementet.